internal/init/init.go

```diff
@@ -90,38 +91,53 @@ func (c *Client) NewCommand() *cobra.Command {
 			c.UserService = *metalGoClient.UsersApi
 			c.ProjectService = *metalGoClient.ProjectsApi
 
-			include := []string{} // []string | Nested attributes to include. Included objects will return their full attributes. Attribute names can be dotted (up to 3 levels) to included deeply nested objects. (optional)
-			exclude := []string{"devices", "members", "memberships", "invitations", "ssh_keys", "volumes", "backend_transfer_enabled", "updated_at", "customdata", "event_alert_configuration",
-				"timezone", "features", "avatar_url", "avatar_thumb_url", "two_factor_auth", "mailing_address", "max_projects", "verification_stage", "emails", "phone_number", "restricted",
-				"full_name", "email", "social_accounts", "opt_in_updated_at", "opt_in", "first_name", "last_name", "last_login_at"}
-			user, _, err := c.UserService.FindCurrentUser(context.Background()).Include(include).Exclude(exclude).Execute()
+			var defaultOrgId, defaultProjectId string
+			project, isProjectToken, err := getDefaultProjectOK(c.ProjectService)
 			if err != nil {
 				return err
 			}
-			organization := user.GetDefaultOrganizationId()
-			project := user.GetDefaultProjectId()
-			fmt.Printf("Organization ID [%s]: ", organization)
+			if project != nil {
+				defaultProjectId = project.GetId()
+				defaultOrgId = project.Organization.GetId()
+			}
+
+			if !isProjectToken {
+				// API token provided is user token
+				include := []string{} // []string | Nested attributes to include. Included objects will return their full attributes. Attribute names can be dotted (up to 3 levels) to included deeply nested objects. (optional)
```

Comment:
We can take out the `.Include()` entirely if we're not sending any include params.